src/main/java/gov/nasa/pds/validate/ri/CommandLineInterface.java

```diff
@@ -18,18 +20,17 @@
 
 public class CommandLineInterface {
   final private Logger log = LogManager.getLogger(CommandLineInterface.class);
-  private long broken = -1, total = -1;
   final private Options opts;
+  private long broken = -1, total = -1;
+  private Map<String, List<String>> duplicates = null;
 
   public CommandLineInterface() {
     super();
     this.opts = new Options();
-    
-    // Disabling this argument for the time being since the Search API does not yet support authorized access
+    /* registry is not ready. Deactivating this option until it becomes ready
```

Comment:
Thank you, thank you, **thank you** for including a comment explaining _why_ something is being commented-out! 🎉